src/components/Calculator.js

```diff
@@ -1,42 +1,72 @@
 import React from 'react';
-import Button from './Button';
 import './Calculator.css';
+import calculate from '../logic/calculate';
+import Output from './Output';
+import Button from './Button';
+
+const generateResult = ({ total, next, operation }) => {
+  if (!total && !next && !operation) {
+    return 0;
+  }
+  if (!total && next) {
+    return `${next}`;
+  }
+  if (operation && total && !next) {
+    return `${total} ${operation}`;
+  }
+  if (total && next && operation) {
+    return `${total} ${operation} ${next}`;
+  }
+  return `${total}`;
+};
 
 class Calculator extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      total: 0,
+      next: 0,
+      operation: null,
+    };
   }
 
+  clickEvent = (event) => {
+    const obj = this.state;
+    console.log(obj);
```

Comment:
- [ ] Do not commit `console.log` to your repo. It’s ugly, it kills performance, and in the interest of security, it can make confidential data be visible to anyone using the browser tools to look at your website.